### ISSUE TITLE
Strange behavior YTPPlayNext starting with 3.1.11

### ISSUE THREAD

**Issue by ak545** [pupunzi/jquery.mb.YTPlayer#320]
Hi!

Starting with version 3.1.11 when using the video list, the YTPPlayNext method behaves strangely.
Instead of the expected video change, there is an endless switching without playing.
Prior to version 3.1.9 inclusive, everything worked well.

I do not understand, or I'm doing something wrong, or something is wrong with YTPlayer ...
Your example _demo_videoList_autoChange.html_ behaves exactly the same (it also has errors with YTPPlayNext)

Please check, maybe there is an error in YTPPlayNext.
Perhaps this is somehow related to the problem that was fixed in version 3.1.10.

Sorry for my bad English.
In order to understand the essence of the problem, see the code example below:

```

<!DOCTYPE html>
<html>
<head>
...

<link href="/css/jquery.mb.YTPlayer.min.css?v=3.1.9" media="all" rel="stylesheet" type="text/css">


<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script src="/js/playlist2.js</script>


<!-- YTPPlayNext WORKING -->
<script src="/js/3.1.9/jquery.mb.YTPlayer.js"></script>

<!-- YTPPlayNext NOT WORKING -->
<!--<script src="/js/3.1.11/jquery.mb.YTPlayer.js"></script>-->


<script src="/js/apikey.js"></script>
<script src="/js/magister.js?v=0.17"></script>
...
</head>
<body>

<a id="bgndVideo" class="player"></a>

...

<button id="vnext">Next</button><br>
<span id="vtime"></span><br>
<span id="vname"></span><br>

</body>
</html>



// playlist2.js
var videos = [
	{
		videoURL:"c8aFcHFu8QM",
		containment:"body",
		autoPlay:true,
		mute:false,
		startAt:0,
		loop:false,
		quality:"default"
	},
	{
		videoURL:"68zAq1_Dj1o",
		containment:"body",
		autoPlay:true,
		mute:false,
		startAt:0,
		loop:false,
		quality:"default"
	},
	{
		videoURL:"mgj-0fNjaBA",
		containment:"body",
		autoPlay:true,
		mute:false,
		startAt:0,
		loop:false,
		quality:"default"
	}
]



// magister.js
var myPlayListPlayer; 

function changeVideo() {
    if (myPlayListPlayer.length) {
        myPlayListPlayer.YTPPlayNext();
    }
}


$(function() {
	myPlayListPlayer = $('#bgndVideo');

	myPlayListPlayer.YTPlaylist(videos, true, function(video) {
		// any...
	});
	
        myPlayListPlayer.on("YTPTime", function(e) {
            var currentTime = e.time;
            var prog = myPlayListPlayer.YTPManageProgress();
            var _ct = prog.currentTime,
                _dur = prog.totalTime;
            if (_ct !== undefined) {
                if (!isNaN(_ct)) {
                    _ct = $.mbYTPlayer.formatTime(_ct);
                    _dur = $.mbYTPlayer.formatTime(_dur);
                }
                if ($('#vtime').length) {
                    $("#vtime").html('_ct + ' / ' + _dur);
                }
            }
        });

        myPlayListPlayer.on("YTPData", function(e) {
            CURRENT_VIDEOID = e.prop.id;
            if ($('#vname').length) {
                $('#vname').html('e.prop.title);
            }
        });


        $("#vnext").click(function() {
            changeVideo();
            return false;
        });
});


```

**Comment by pupunzi** [comment]
This bug has been fixed with the 3.1.12 release.

**Comment by ak545** [comment]
Thanks!
